gateway-service-definitions/src/main/resources/services/hue/1.0.0/rewrite.xml

```diff
@@ -0,0 +1,134 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<rules>
+  <rule dir="IN" name="HUE/hue/inbound/huerule" flow="OR">
+    <!-- We have exceptions for dashboard/hbase/notebook, because endpoint that ends at the app/lib level need to terminate with a slash and we remove them in generic case. -->
+    <!-- Other solution would be to accept trailing slash on all endpoints, but there's some issue with the filebrowser -->
+    <match pattern="*://*:*/**/hue/hue/dashboard?{**}">
```

Comment:
I think this will be `/hue/hue` in the browser so might be a bit weird. I think I follow the comment, but just calling out in case there is something we can do to make this easier.

Reply:
/hue/hue is unfortunate, but I don't see any other solutions. We have endpoints on both /hue/dashboard & /dashboard and this seems the only way to disambiguate both.